webview/src/experiments/components/table/Errors.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
```

Comment:
[Q] Better placement for this? Should it be a shared component? We will probably use something very similar for plots.

Reply:
We can move it in the shared folder when it's used in both webviews.